README.md

```diff
@@ -154,6 +155,17 @@ Note: Compose project names default to the name of the directory which contains
 
 If all else fails, this script includes a `--help` option which explains what each of the options do and how they are supposed to be used.
 
+## Using static images with released iRODS versions
+
+`--use-static-images` enables building and running a Docker image with iRODS packages installed rather than downloading an installing the packages at runtime. This is useful for testing plugins against released versions of iRODS or for quickly reproducing issues on past versions.
```

Comment:
"downloading an installing" -> "downloading and installing"

Reply:
Done

Reply:
Done